lib/alephant/broker/component_meta.rb

```diff
@@ -21,6 +21,18 @@ def component_key
         "#{id}/#{opts_hash}"
       end
 
+      def as_json
```

Comment:
yeah, I couldn't really come up with a good name to be honest, so I followed the ActiveSupport `as_json`, `to_json` pattern...  Will try to think of something more appropriate. 👍